x-pack/plugins/security_solution/public/detection_engine/rule_response_actions/response_actions_form.tsx

```diff
@@ -9,17 +9,12 @@ import React, { useEffect, useMemo, useState } from 'react';
 import { EuiCallOut, EuiSpacer } from '@elastic/eui';
 import { map, reduce, upperFirst } from 'lodash';
 import ReactMarkdown from 'react-markdown';
-import { css } from '@emotion/react';
 import { ResponseActionsWrapper } from './response_actions_wrapper';
 import { FORM_ERRORS_TITLE } from '../../detections/components/rules/rule_actions_field/translations';
 import { ResponseActionsHeader } from './response_actions_header';
 import type { ArrayItem, FormHook } from '../../shared_imports';
 import { useSupportedResponseActionTypes } from './use_supported_response_action_types';
 
-const FieldErrorsContainer = css`
```

Comment:
Could you use `StyledComponent` instead?

Reply:
Yes, styledComponent worked, but really didn't bring any change to UI. So I just removed the whole thing ;p